server/common/admin-rest-client/src/main/java/org/kaaproject/kaa/server/common/admin/AdminClient.java

```diff
@@ -527,6 +528,18 @@ public void editUserConfiguration(EndpointUserConfigurationDto endpointUserConfi
         restTemplate.postForLocation(restTemplate.getUrl() + "userConfiguration", endpointUserConfiguration);
     }
 
+    public EndpointSpecificConfigurationDto editEndpointSpecificConfiguration(EndpointSpecificConfigurationDto configuration) throws Exception {
+        return restTemplate.postForObject(restTemplate.getUrl() + "endpointConfiguration", configuration, EndpointSpecificConfigurationDto.class);
+    }
+
+    public void deleteEndpointSpecificConfiguration(String endpointKeyHash) throws Exception {
```

Comment:
We need to add few more methods - ability to delete, update and find configuration by endpointKeyHash and specific version.